docs/howto/garbage-collection-index.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Clicking on the index brings me to the `committed gc` page.

Reply:
It's a feature :)
Wanted to leave the default page as it is today, without an actual index page.